tests/pql/test_follows.py

```diff
@@ -0,0 +1,67 @@
+import pytest
+
+from ats.pql.pql import parse_query
+
+
+def test_relation_follows_in_query():
+    result = parse_query(
+        """ stmt s1;
+            Select s1 such that Follows(s1, 3)
+           """
+    )
+
+    assert result[0]["relations"][0]["relation"] == "Follows"
+
+
+def test_parameters_relation_follows_in_query():
+    result = parse_query(
+        """ stmt s1;
+            Select s1 such that Follows('_', 10)
+           """
+    )
+
+    assert result[0]["relations"][0]["parameters"][0] == "'_'"
```

Comment:
zastanawiam sie czy to nie powinno zwracac np: `None` albo jakiegos konkretnego obiektu do ktorego pozniej mozna sie porownywac